clang-tools-extra/test/clang-tidy/checkers/performance/for-range-copy.cpp

```diff
@@ -234,6 +239,22 @@ void positiveOnlyAccessedFieldAsConst() {
   }
 }
 
+void positiveOnlyAccessedFieldAsConstBinding() {
```

Comment:
Why `AccessedField` here, it seems to be `JustUsedAsConst` as in other test names?

Reply:
Done.